### PR TITLE
Added package for mixed effects regression analysis

### PR DESCRIPTION
**Pull request by claudio525**

### CODE REVIEW

**Review by jpa198 — APPROVED** [review]

**Review by lee-robin — APPROVED** [review]

File: mera/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+This package contains two different implementations for running mixed-effects regression analysis for ground motion prediction models with 2 random effects, event and site.
+
+The only difference between the two implementations is the underlying packages they use.
+
+---- 
+
+### Installation
+
+Navigate to the `Ancillar-Tools` directory and run `pip install -e ./mera`
```
Comment (jasonmotha) [inline mera/README.md:11]: Ancillary-Tools*

File: mera/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Note: Dependencies have to be installed manually depending on which implementation is used
+
+-----
+
+### Implementation - statsmodel
```
Comment (jasonmotha) [inline mera/README.md:17]: howver -> However

File: mera/setup.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="mera",
+    version="22.8.1",
+    packages=find_packages(),
+    url="",
+    description="Code for GM prediction mixed-rffects regression analysis",
```
Comment (jasonmotha) [inline mera/setup.py:8]: rffects -> effects

**Review by jasonmotha — COMMENTED** [review]
Have you considered grammarly? :yum: 

Also is it worth having an example script with this package?

**Comment by jpa198** [comment]
Well I guess this is going to hang around for a month...

**Review by jpa198 — APPROVED** [review]

**Pull request merged**